lib/Backend/arm64/LowerMD.cpp

```diff
@@ -1408,17 +1398,19 @@ LowererMD::LowerEntryInstr(IR::EntryInstr * entryInstr)
         }
     }
 
-    Assert(spSaveOffset == regSaveArea);
-
     if (hasTry)
     {
         // Copy the value of SP before we allocate the locals area. We'll save this value on the stack below.
-        Lowerer::InsertMove(
-            IR::RegOpnd::New(nullptr, EH_STACK_SAVE_REG, TyMachReg, this->m_func),
-            IR::RegOpnd::New(nullptr, RegSP, TyMachReg, this->m_func),
-            insertInstr);
+        IR::RegOpnd* ehReg = IR::RegOpnd::New(nullptr, EH_STACK_SAVE_REG, TyMachReg, this->m_func);
```

Comment:
>EH_STACK_SAVE_REG [](start = 55, length = 17)

what's the value in EH_STACK_SAVE_REG at this point?
I original code SP value was copied in EH_STACK_SAVE_REG at this point.